server/sonar-server/src/test/java/org/sonar/server/computation/step/ComputeQProfileMeasureStepTest.java

```diff
@@ -133,25 +134,6 @@ public void nothing_to_add_when_measure_already_exists_on_project() throws Excep
     assertThat(measureRepository.getAddedRawMeasures(PROJECT_REF)).isEmpty();
   }
 
-  @Test
-  public void nothing_to_add_when_no_qprofile_computed_on_project() throws Exception {
-    treeRootHolder.setRoot(MULTI_MODULE_PROJECT);
-
-    underTest.execute();
-
-    assertThat(measureRepository.getAddedRawMeasures(PROJECT_REF)).isEmpty();
-  }
-
-  @Test
-  public void nothing_to_add_when_qprofiles_computed_on_project_are_empty() throws Exception {
```

Comment:
You should keep this test by testing that measure is empty when qprofile data from AnalysisMetadataHolderRule is empty